src/deploy/NVA_build/NooBaa.Dockerfile

```diff
@@ -33,6 +33,14 @@ RUN tar \
 
 #####################################################################################################################################
 
+# Do not install docs and multiple langaugages.
+LANG="en_US"
+echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
+
+awk '(NF==0&&!done){print "override_install_langs='$LANG'\ntsflags=nodocs";done=1}{print}' \
+    < /etc/dnf/dnf.conf > /etc/dnf/dnf.conf.new
+mv /etc/dnf/dnf.conf.new /etc/dnf/dnf.conf
+
```

Comment:
should move to line 51: https://github.com/noobaa/noobaa-core/pull/5857/files#diff-e260bb3018701cb64c7b23e88c7fc672R51

In order for the changes to affect it should be on the last stage.

Reply:
should be between:

FROM centos:8

ENV container docker